src/common.rs

```diff
@@ -1572,10 +1572,49 @@ pub fn display_access_key_list(access_keys: &[near_primitives::views::AccessKeyI
     table.printstd();
 }
 
+/// sort network names based on account_id if provided
+/// if account_id is not provided, sort based on network name
+/// this is to make sure that the default network is always the same
+/// otherwise, sort the network names which maps to the same account_id to the top of the list,
+/// and then sort the rest of the network names alphabetically
 pub fn input_network_name(
     config: &crate::config::Config,
+    account_id: Option<crate::types::account_id::AccountId>,
```

Comment:
I don't see a reason to pass `account_id` by value, so I suggest passing it by reference to avoid unnecessary clones on the caller side:

```suggestion
    account_id: Option<&crate::types::account_id::AccountId>,
```